app/views/catarse_bootstrap/shared/_footer_big.html.slim

```diff
@@ -21,6 +21,7 @@ footer.main-footer.main-footer-neg
   .w-container
     .footer-full-copyleft 
       = image_tag "logo-footer.png", class: "u-marginbottom-20"
-      .lineheight-loose Feito com amor | 2015 | Open source
-      
-      
\ No newline at end of file
+      .lineheight-loose
+        |  Feito com amor |
+        |  2015 |
+        a.link-footer-inline< href="#{CatarseSettings[:github_url]}" Open source
```

Comment:
I would rather use the entire sentence as a link. So we wouldn't need to worry with the link being inline.
```slim
a.link-footer[href="#{CatarseSettings[:github_url]}"] Feito com amor | #{Date.today.year} | Open source
```